docs/framework/tutorial/shard.md

```diff
@@ -0,0 +1,353 @@
+---
+short-description: Refining and packaging the downloaded content
+...
+# Creating a shard #
+
+In this section, we'll view the blog posts we've ingested in the previous section, and make some improvements to our ingester.
+We'll add some customizations to the visual theme, and make sure videos are downloaded for offline viewing too.
+Then we will package the hatch for distribution, in an archive format called a "shard".
+
+We'll use Hatch Previewer to view the blog posts.
+If you haven't installed it yet, check how to do so in the [Install](tutorial/install.md#installing-hatch-previwer) section.
+
+## Viewing the blog posts ##
+
+Running the ingester in the last section should have created a directory with a name like `hatch_cc-blog_20180215_104946` (where the numbers are the date and time that the ingester was run.)
+Start Hatch Previewer, either by clicking its icon on the desktop, or from the command line:
+```bash
+flatpak run com.endlessm.HatchPreviewer
+```
+You'll be shown a dialog box where you select a directory.
+Navigate to the hatch directory and select it.
+
+> **NOTE:** You can also pass the hatch directory on the command line, but you have to give an absolute path due to a limitation in Flatpak.
```

Comment:
Really? Why is that out of curiosity?

Reply:
For file-type command-line arguments, flatpak opens a portal for the file if the app doesn't have filesystem access, and the portal code is limited to absolute paths for some reason.